packages/@lwc/engine-server/src/__tests__/fixtures.spec.ts

```diff
@@ -103,11 +103,22 @@ function testFixtures(options?: RollupLwcOptions) {
             pattern: '**/index.js',
         },
         async ({ filename, dirname, config }) => {
-            const compiledFixturePath = await compileFixture({
-                input: filename,
-                dirname,
-                options,
-            });
+            let compiledFixturePath;
+
+            try {
+                compiledFixturePath = await compileFixture({
+                    input: filename,
+                    dirname,
+                    options,
+                });
+            } catch (err: any) {
+                // Filter out the stacktrace, just include the LWC error message
+                const message = err?.message?.match(/(LWC\d+[^\n]+)/)?.[1];
+                return {
+                    'expected.html': '',
+                    'error.txt': message ?? '',
```

Comment:
`err.message` does include the stack here because the error actually comes from Babel-via-Rollup.

Also if `err?.message` is nullish, then `message` will be nullish as well.